src/plans/ReferenceMission1.plp

```diff
@@ -72,7 +72,11 @@ ReferenceMission1: Concurrence
       LibraryCall StartSampleAnalysis;
       Wait 2;  // Wait for (stubbed) sample analysis to finish.
     }
-    else log_error ("Failed to find ground, aborting.");
+    else
+    {
+      log_error ("Failed to find ground, aborting.");
+      LibraryCall Stow();
```

Comment:
Good addition!  I have a Jira for restowing in _all_ situations, but this is fine.

Reply:
I know, this is why i mentioned it briefly touches on the issue but still I did want to include this change as it really bugs me a to see the arm in the position .. also I haven't yet tested but this should allow us to rerun the reference mission again without having to restart the simulation

Reply:
Just for reference, I did try stopping and re-running the reference mission 1 with my change and the arm is works according to the plan again. The catch is the antenna since it wasn't reset to its default position the plan re-run doesn't seem to have impact. I am going to update the ticket 510 to include this.

Reply:
Actually, let's discuss this first during the meeting today

Reply:
Shouldn't the arm go to the Unstow position before Stow? I thought that was the process we were using.

Reply:
This library plan (Stow.plp) does the unstow() command prior to stow().   If you think this requirement should be exposed to the user, we can change this under a new issue.

Reply:
@mogumbo I confirm what Mike side. The command internally goes into unstow then stow